Tools/CISupport/ews-build/steps.py

```diff
@@ -2430,8 +2430,9 @@ def start(self):
             # For build-only bots, the expectation is that tests will be run on separate machines,
             # so we need to package debug info as dSYMs. Only generating line tables makes
             # this much faster than full debug info, and crash logs still have line numbers.
+            # Some projects (namely lldbWebKitTester) require full debug info, and may override this.
             self.setCommand(self.command + ['DEBUG_INFORMATION_FORMAT=dwarf-with-dsym'])
-            self.setCommand(self.command + ['CLANG_DEBUG_INFORMATION_LEVEL=line-tables-only'])
+            self.setCommand(self.command + ['CLANG_DEBUG_INFORMATION_LEVEL=$(WK_OVERRIDE_DEBUG_INFORMATION_LEVEL:default=line-tables-only)'])
```

Comment:
How does this work with CMake?

Reply:
This code is Xcode-only. Just outside the diff context is an `if platform in ('mac', 'ios', 'tvos', 'watchos')` statement.